{"name":"Solicitudes concurrentes para clientes","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"El cliente no disponible CUSTOMER1 no fue encontrado en la respuesta.","trace":"java.lang.AssertionError: El cliente no disponible CUSTOMER1 no fue encontrado en la respuesta.\r\n\tat org.junit.Assert.fail(Assert.java:89)\r\n\tat org.junit.Assert.assertTrue(Assert.java:42)\r\n\tat co.appgate.stepdefinitions.ValidacionDeUsuarios.validarClienteDisponible(ValidacionDeUsuarios.java:268)\r\n\tat co.appgate.stepdefinitions.ValidacionDeUsuarios.elSistemaRespondeQueEstaDisponibleYNoEstanDisponibles(ValidacionDeUsuarios.java:225)\r\n\tat ✽.el sistema responde que \"CUSTOMER1\" está disponible y \"CUSTOMER2, CUSTOMER3, y CUSTOMER4\" no están disponibles(file:///C:/Users/valen/Documents/RabbitRest/src/test/resources/features/ValidacionDeUsuarios.feature:10)\r\n"},"stage":"finished","description":"  Validar si un cliente está disponible para el servicio de detección de phishing.","steps":[{"name":"Given  que el servicio RabbitMQ está corriendo","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1735838281909,"stop":1735838284350},{"name":"And  la base de datos PostgreSQL está activa","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1735838284350,"stop":1735838284568},{"name":"When  se realizan solicitudes simultáneas para \"CUSTOMER1\", \"CUSTOMER2\", \"CUSTOMER3\", \"CUSTOMER4\"","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1735838284570,"stop":1735838285814},{"name":"Then  el sistema responde que \"CUSTOMER1\" está disponible y \"CUSTOMER2, CUSTOMER3, y CUSTOMER4\" no están disponibles","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"El cliente no disponible CUSTOMER1 no fue encontrado en la respuesta.","trace":"java.lang.AssertionError: El cliente no disponible CUSTOMER1 no fue encontrado en la respuesta.\r\n\tat org.junit.Assert.fail(Assert.java:89)\r\n\tat org.junit.Assert.assertTrue(Assert.java:42)\r\n\tat co.appgate.stepdefinitions.ValidacionDeUsuarios.validarClienteDisponible(ValidacionDeUsuarios.java:268)\r\n\tat co.appgate.stepdefinitions.ValidacionDeUsuarios.elSistemaRespondeQueEstaDisponibleYNoEstanDisponibles(ValidacionDeUsuarios.java:225)\r\n\tat ✽.el sistema responde que \"CUSTOMER1\" está disponible y \"CUSTOMER2, CUSTOMER3, y CUSTOMER4\" no están disponibles(file:///C:/Users/valen/Documents/RabbitRest/src/test/resources/features/ValidacionDeUsuarios.feature:10)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1735838285814,"stop":1735838285965}],"attachments":[],"parameters":[],"start":1735838281902,"stop":1735838285983,"uuid":"ed247bd1-e32f-4a1f-a31b-110e3737d07b","historyId":"3ec2e9af468c60667826fb1df283ab","fullName":"Validación de Disponibilidad de Cliente: Solicitudes concurrentes para clientes","labels":[{"name":"host","value":"VandV"},{"name":"thread","value":"21144@VandV.Test worker(1)"},{"name":"feature","value":"Validación de Disponibilidad de Cliente"},{"name":"story","value":"Solicitudes concurrentes para clientes"},{"name":"suite","value":"Validación de Disponibilidad de Cliente"},{"name":"testClass","value":"Solicitudes concurrentes para clientes"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///C:/Users/valen/Documents/RabbitRest/src/test/resources/features/ValidacionDeUsuarios.feature"},{"name":"package","value":"src.test.resources.features.ValidacionDeUsuarios_feature.Validación de Disponibilidad de Cliente"}],"links":[]}